app/views/publishers/_choose_channel_type.html.slim

```diff
@@ -11,7 +11,7 @@
         h6= t "publishers.choose_new_channel_type.website.heading"
         = t "publishers.choose_new_channel_type.website.description"
 
-= link_to(publisher_register_youtube_channel_omniauth_authorize_path, class: 'card block-link--card') do
+= link_to(publisher_register_youtube_channel_omniauth_authorize_path, class: 'card block-link--card', :"data-piwik-action" => "AddYoutubeChannelClicked", :"data-piwik-name" => "Clicked", :"data-piwik-value" => "AddChannelFlow") do
```

Comment:
@alexwykoff Heads up, the clicking of this button does not correlate with channel creation, only beginning the flow.